src/components/screens/Service.tsx

```diff
@@ -76,6 +76,7 @@ export const Service = ({
     (item: TServiceTile, index: number) => (
       <ServiceTile
         draggableId={umlautSwitcher(item.title) || umlautSwitcher(item.accessibilityLabel)}
+        draggableKey={index + (item.title || item.accessibilityLabel)}
```

Comment:
let's use `item${item}-index${index}` here as well with `title` or `accessibilityLabel` as `item`?

Reply:
Yes, that's right. I didn't want to change it just because the `key` prop is used in this format.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/6090ffdf266c658530e01696e576e549ef9a5065